README.md

```diff
@@ -56,13 +56,24 @@ sudo dnf install -y glibc-static
 make
 ```
 
+### Environment Configuration
+
+Before running Microshift, the host must first be configured.  This can be handled by running  
+
+`MICROSHIFT_CONFIG_ENV_ONLY=true ./install.sh`
+
+Or remotely as 
+
+`curl -sfL https://raw.githubusercontent.com/redhat-et/microshift/main/install.sh |  MICROSHIFT_CONFIG_ENV_ONLY=true sh -`
+
 ### Running
 
-Use `install.sh` to set up your sytem and install run-time dependencies for Microshift, then simply:
 ```
 sudo microshift run
 ```
 
+Developers:
+
```

Comment:
Is this necessary, given that we're already in the "Developing MicroShift" section?